### ISSUE TITLE
Unable to add a Pool in the root of the pipeline

### ISSUE THREAD

**Issue by thomhurst** [sharpliner/sharpliner#224]
In my existing pipelines, I have a pool in the root of the yaml like this:

```yaml
trigger:
  batch: true
  branches:
    include:
    - develop

pool:
  name: 'my-hosted-ubuntu'

variables:
- template: variables/vars.yml
```

I can't see anyway of doing this currently, since this are the only things I can set:
![image](https://user-images.githubusercontent.com/30480171/186900871-1dac61d8-12ee-4daa-b8f4-2d3cd85fff7c.png)

**Comment by premun** [comment]
Thanks for logging this, fix is here - https://github.com/sharpliner/sharpliner/pull/226
I will publish a new version shortly.

**Comment by premun** [comment]
This fix will be in a version `1.3.7` in about 15 minutes
